tests/snmpsim/sophos4.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.1.0|4|Linux g4dc0e52 
```

Comment:
Thanks for the regex @laf  I was thinking something like this but gave me a good head start :)